weave-js/src/components/PagePanelComponents/Home/Browse3/context.tsx

```diff
@@ -428,7 +475,11 @@ type RouteType = {
     wfTable?: WFDBTableType
   ) => string;
   entityUrl: (entityName: string) => string;
-  projectUrl: (entityName: string, projectName: string) => string;
+  projectUrl: (
+    entityName: string,
+    projectName: string,
+    noEncode?: boolean
+  ) => string;
   typeUIUrl: (
     entityName: string,
     projectName: string,
```

Comment:
Can't we assume entityName and projectName are [always safe](https://github.com/wandb/core/blob/master/services/gorilla/validate.go#L30)?